addons/account/tests/test_bad_reconciliation.py

```diff
@@ -0,0 +1,268 @@
+            [('user_type_id', '=', self.env.ref(
+                'account.data_account_type_expenses').id),
+             ('company_id', '=', self.company.id)],
+            limit=1)
+        self.account_revenue = self.env['account.account'].search(
+            [('user_type_id', '=', self.env.ref(
+                'account.data_account_type_revenue').id),
+             ('company_id', '=', self.company.id)],
+            limit=1)
+
+        self.cc_bank_account = self.env['account.account'].create({
+            'name': 'CC Bank Account',
+            'code': '9991',
+            'company_id': self.company.id,
+            'user_type_id': self.env.ref(
+                'account.data_account_type_current_assets').id
+
+        })
+        self.company.income_currency_exchange_account_id = self.env[
+            'account.account'].create({
+                'name': 'Exchange Profit',
+                'code': '9993',
+                'company_id': self.company.id,
+                'user_type_id': self.env.ref(
+                    'account.data_account_type_revenue').id
+            })
+        self.company.expense_currency_exchange_account_id = self.env[
+            'account.account'].create({
+                'name': 'Exchange Loss',
+                'code': '9994',
+                'company_id': self.company.id,
+                'user_type_id': self.env.ref(
+                    'account.data_account_type_expenses').id
+            })
+        self.currency_cc = self.env['res.currency'].with_context(
+            company_id=self.company.id,
+            force_company=self.company.id).create({
+                'name': 'CC',
+                'rounding': 0.010000,
+                'symbol': 'CC',
+                'position': 'after',
+            })
+        self.customer = self.env['res.partner'].create({
+            'name': 'Test customer',
+            'customer': True,
+        })
+        self.supplier = self.env['res.partner'].create({
+            'name': 'Test supplier',
+            'customer': True,
+        })
+        self.cc_journal = self.env['account.journal'].create({
+            'name': 'CC Payments Journal',
+            'code': 'CC',
+            'type': 'bank',
+            'company_id': self.company.id,
+            'default_debit_account_id': self.cc_bank_account.id,
+            'default_credit_account_id': self.cc_bank_account.id,
+        })
+        # We want to make sure that the company has no rates, that would
+        # screw up the conversions.
+        company_rates = self.env['res.currency.rate'].search(
+            [('currency_id', '=', self.company.currency_id.id)])
+        company_rates.unlink()
+
+        return res
+
+    def _check_account_balance(self, account):
+        balance = sum(self.env['account.move.line'].search(
+            [('account_id', '=', account.id)]).mapped('balance'))
+        return balance
+
+    def test_01(self):
+        """ Implements a successful scenario """
```

Comment:
what successful scenario? is it new compared to the tests in test_reconciliation.py? if yes, it should be
* moved there
* be more explicit on what it tests

Reply:
My opinion is: I think this use case is already tested, so it should be removed from the PR